core/server/api/v2/utils/serializers/input/utils/url.js

```diff
@@ -0,0 +1,67 @@
+const {absoluteToRelative, getBlogUrl, STATIC_IMAGE_URL_PREFIX} = require('../../../../../../services/url/utils');
+
+function handleImageUrl(imageUrl) {
+    let blogUrl = getBlogUrl().replace(/^http(s?):\/\//, '').replace(/\/$/, '');
```

Comment:
All of these `let`'s can be const's ;) Don't remember if we decided on anything specific in this topic

Reply:
Decision was constantly const in server-side code.

Jury's still out on client-side because everything there is a variation of liberal let (no const outside module scope) so it would be a big change to re-do everything.